internal/code/imports_test.go

```diff
@@ -40,6 +40,6 @@ func TestNameForDir(t *testing.T) {
 
 	assert.Equal(t, "tmp", NameForDir("/tmp"))
 	assert.Equal(t, "code", NameForDir(wd))
-	assert.Equal(t, "internal", NameForDir(wd+"/.."))
```

Comment:
Apologies for failing to sanity check the tests initially.

Turns out  that the assumption behind the this assertion broke due to moving `tools.go`. I've pointed it at a different path that has no Go code. This seems to match the intent behind the original test for the `internal/code/..` path.

I _may_ have misunderstood, so happy for alternative suggestions. :thinking: